pkg/auth/providers/saml/saml_provider.go

```diff
@@ -278,6 +290,19 @@ func (s *Provider) isThisUserMe(me v3.Principal, other v3.Principal) bool {
 	return false
 }
 
+func (s *Provider) CanAccessWithGroupProviders(userPrincipalID string, groupPrincipals []v3.Principal) (bool, error) {
+	config, err := s.getSamlConfig()
+	if err != nil {
+		logrus.Errorf("Error fetching saml config: %v", err)
```

Comment:
is this err not going to bubble up?

Reply:
this isn't actually any of my changes, the func is the original from the file when i got it just moved around or something and shows as my change. i think as a top level auth provider it will so ya this is redundant a bit but im not in a place to make the call